doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -15,6 +15,7 @@
 * [Ordering between:$
  different attribute value types](#ordering-between-different-attribute-value-types)
 * [Initial notifications](#initial_notifications)
+* [Oneshot Subscription](#oneshot-subscriptions)
```

Comment:
Better use underscore (`_`) to use the same style than the others.

Reply:
fixed in 5c2a50e